version.sbt

```diff
@@ -1 +1,17 @@
+/*
```

Comment:
With the `release.yml` file, this file cannot be present.

Reply:
Yes that's right. Please take a closer look at https://github.com/sbt/sbt-ci-release#sbt, particularly this section:

> Make sure build.sbt does not define any of the following settings
>   version: handled by sbt-dynver
>   publishTo: handled by sbt-ci-release
>   publishMavenStyle: handled by sbt-ci-release
>   credentials: handled by sbt-sonatype

Reply:
still outstanding ^